sql_request_abstract/models/sql_request_mixin.py

```diff
@@ -242,6 +242,8 @@ def _check_execution(self):
             self.env.cr.execute(query)
             res = self._hook_executed_request()
         except ProgrammingError as e:
+            # it helps a lot for debugging to have raw query string without \n
+            print(query)
```

Comment:
Why not emit it as logger.warning or similar?

Reply:
logger'll display `\n` not return to the line when occurs, isn't it ?

Reply:
I don't think so if you pass directly the query, or use `from pprint import pprint; pprint(query)`